osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -130,8 +131,27 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             ((IHasText)displayedCountSpriteText).Text = formatCount(Current.Value);
+            ((IHasText)popOutCount).Text = formatCount(Current.Value);
 
             Current.BindValueChanged(combo => updateCount(combo.NewValue == 0), true);
+
+            // Since layout depends on combo font height we need to update it during skin change
+            skin.SourceChanged += updateLayout;
```

Comment:
How were you noticing this? Changing skins generally reloads all components, no? Replacing this with a single call in `LoadComplete` to `updateLayout` seems to still work fine for me.

Reply:
That was my intuition, hasn't crossed my mind that entire stack would be reinitialized on legacy skin change. Will retest later

Reply:
Yeah, it gets reinitialized. Removed skin dependency